### DISCUSSION TITLE
Watching YouTube videos on Mac OS 9

### DISCUSSION THREAD

**Discussion by Bluescreen92jrd** [atauenis/webone#112] [General]
Hello everyone,

I've found out that I _might,_ emphasis on **might,** be able to use Windows Media Player 7/RealPlayer 8 Plus/QuickTime 6 to try playing YouTube videos in the proxy's embedded video player. However, I'm not sure about the conversion options. What I've found out is this:

- Sometimes, when QuickTime is used, I only get the video without sound.
- Windows Media Player will say that the playlist is invalid when I attempt to play an ASF stream.

Do you have any suggestions as to what video/audio formats to use for Mac OS 9, or if you wish to go that route, what do you think will work best for Mac OS X Leopard? I'm looking for any possible advice.

**Comment by way5** [comment]
Hi @Bluescreen92jrd . 
Sorry about late reply, though not sure yet how to answer your question right away. It requires testing. At this very moment I have a couple of machines for testing but neither of them has MacOS 9 nor Leopard. (MacOS 8 and Tiger)

My suggestion is to wait till someone got a chance to test in properly (taking in account mentioned env.) or provide some more data on a particular issue.

~ Thanks

**Comment by atauenis** [comment]
At this moment, WebOne is using combination of YT-DLP and FFMPEG applications to process videos. So it's possible to test the converting options by experimenting with manually running them via command prompt, then open output files in different players.

With SheepShaver emulator it's possible to emulate OS 7.5-9.0 (and 9.1 with patched 9.0 kernel), and it have possibility to exchange file with host PC. But it is a bit difficult as it giving access to all drives on host PC. I'm don't know any emulators which can emulate later PPC systems with OSX on Windows hosts, and with proper file exchange feature. VirtualBox can emulate any x86/x64 OSX, such as Tiger, Leopard or even Mojave, but have no file exchange feature.

Also there is an format which is working on any systems - Shockwave Flash with MJPEG+MP3. It have a visible quality loss (need to set up bitrate, which is not currently implemented in WebOne - need some help here from FFMPEG gurus), but is working on any browser with Flash Player 3.0 or newer (Netscape, Mozilla, Opera, MSIE). At least on Windows & OS/2 (not tested on Mac at this time, but probably should too).
